src/WebJobs.Script/Host/PrimaryHostCoordinator.cs

```diff
@@ -203,7 +203,10 @@ private void ResetLease()
 
         private void SetTimerInterval(TimeSpan interval, TimeSpan? dueTimeout = null)
         {
-            _timer.Change(dueTimeout ?? interval, interval);
+            if (!_disposed)
+            {
+                _timer.Change(dueTimeout ?? interval, interval);
```

Comment:
When debugging, I get an object disposed exception from this code from time to time